Src/ILGPU.Tests/ArrayViews.cs

```diff
@@ -230,6 +230,82 @@ public void ArrayViewLengthInBytes(int length)
             Verify(buffer.View, expected);
         }
 
+        [Fact]
+        public void ArrayViewZeroLengthSubview1d()
```

Comment:
MINOR: Follow existing naming convention - `SubView` and `1D`.
```suggestion
        public void ArrayViewZeroLengthSubView1D()
```